src/io/result_io.jl

```diff
@@ -494,13 +496,14 @@ function setup_result_store!(domain::Domain, scen_spec::DataFrame)::Tuple
     end
     stat_store_names = vcat(dhw_stat_names, wave_stat_names)
 
+    n_group_size::Int = domain.coral_growth.n_group_size
```

Comment:
I know `Int` resolves to the base type but I prefer being explicit all the time: `Int` -> `Int64`